### ISSUE TITLE
Bug: Retour breaks multi language environment

### ISSUE THREAD

**Issue by jschopplich** [distantnative/retour-for-kirby#53]
When using retour (by using I mean copying the plugin into `site/plugins`) no content will be loaded (apparently the entire content of e.g. home.de.txt) nor any other text inside translated pages nor translated strings via the `t()` helper can be used.

It did work again by deleting your plugin. I double checked that.

****

For example visiting: `retour.co.uk/en` throws:
```
path: en
referrer: null
status: failed
pattern: null
```

****

My setup:

de.php
```php
<?php

return [

  'code' => 'de',
  'default' => true,
  'direction' => 'ltr',
  'locale' => 'de_DE.utf8',
  'name' => 'Deutsch',
  'url' => '/',

  'translations' => [
    'home.headline' => '...'
  ]

];
```

en.php
```php
<?php

return [

  'code' => 'en',
  'default' => false,
  'direction' => 'ltr',
  'locale' => 'en_GB.utf8',
  'name' => 'English',
  'url' => 'en',

  'translations' => [
    'home.headline' => '...'
  ]

];
```

**Comment by distantnative** [comment]
Will try to check this on the weekend and hopefully find a fix for it.

**Comment by jschopplich** [comment]
No rush. I‘m sorry I haven‘t provided a reduced test case. Thank you!

**Comment by jschopplich** [comment]
The bug probably occurs because you use `kirby()->root()` before Kirby has actually fully initialized. See https://github.com/getkirby/kirby/issues/1307

**Comment by LeBenLeBen** [comment]
There's seems to be another issue with multilang when you install the plugin. It creates a `site.txt` file in the content folder which breaks the multilang setup and override existing `site.{language-code}.txt`.

**Comment by distantnative** [comment]
@LeBenLeBen It is actually connected to the same problem. The call to the `kirby()` helper initialises another Kirby instance instead of getting the main one - and the second one is without languages. This messes up all kinds of things. I'm currently discussing with Bastian options to fix this issue in the core of Kirby.

**Comment by distantnative** [comment]
Will be fixed with Kirby 3.0.3.
